### PR TITLE
x11-misc/polybar: bump to EAPI=7

### PR DESCRIPTION
**Pull request by end222**
This is a bump to EAPI 7 in the package x11-misc/polybar ebuild. I would also want to become the maintainer of this package since it didn't have one so I have added myself in the metadata.xml file.
Signed-off-by: Pablo Orduna pabloorduna@gmail.com

### CODE REVIEW

File: x11-misc/polybar/metadata.xml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!--maintainer-needed-->
+	<maintainer type="person">
+		<email>pabloorduna98@gmail.com</email>
+		<name>Pablo Orduna</name>
+	</maintainer>
```
Comment (mgorny) [inline x11-misc/polybar/metadata.xml]: Also add proxy-maint to commit for you.

File: x11-misc/polybar/polybar-3.2.1.ebuild [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 PYTHON_COMPAT=( python2_7 )
 
-inherit cmake-utils python-single-r1
+inherit python-single-r1
```
Comment (mgorny) [inline x11-misc/polybar/polybar-3.2.1.ebuild]: cmake-utils is the eclass to use. The EAPI 7 support is work-in-progress, so you just need to wait for it; not work around it by regressing the ebuild.

Comment (end222) [reply x11-misc/polybar/polybar-3.2.1.ebuild]: Understood, I will wait until it supports EAPI 7 and use that eclass.

Thanks you very much for the information

**Comment by monsieurp** [comment]
Thanks for the PR. As pointed out by mgorny. your contribution introduces a regression by not using the cmake eclass and working around the problem. Therefor, we cannot merge it for the time being. If you're still willing to become a maintainer, file a separate PR and we will gladly merge it. Thank you!

**Pull request closed without merging**